### PR TITLE
chore: Update quickstart instructions

### PR DESCRIPTION
**Pull request by rhuss**
for `syndesis` usage + adding a 'rocket launcher' script. Happy about some help for improving the layout and proof reading.

### CODE REVIEW

**Comment by rhuss** [comment]
hey rachel, fancy doing a quick review ? Also happy about suggestions for how to fix the layout (would love to have the screenshots centered and maybe with some drop down shadow).

The original screenshots are below `images/orig`

**Comment by rhuss** [comment]
@kahboom any chance to give this PR a quick review ?

File: content/quickstart.md [hunk 1]
```diff
@@ -17,21 +17,96 @@ toc: true
...
+....
+Sleeping 10s ...
+syndesis-ui-1-kz5qx   0/1       Running   0         34s
+syndesis-ui-1-kz5qx   1/1       Running   0         40s
+syndesis-ui-1-deploy   0/1       Completed   0         46s
+---------------------------------------------------------------------
+Opening http://syndesis.192.168.64.56.nip.io
+```
+
+Here you will be first asked twice to add a Security exception for Minishift's self signed certificate. Please allow this exception:
```
Comment (kahboom) [inline content/quickstart.md]: Is there a reason we are capitalizing Security? Just curious!

Comment (kahboom) [reply content/quickstart.md]: Also should be "self-signed"

File: content/quickstart.md [hunk 2]
```diff
@@ -17,21 +17,96 @@ toc: true
...
+Et voilà, welcome in Syndesisland !
+
+![syndesis](../images/landing_page.png)
+
+
+## Using "syndesis" CLI tool
 
-### Prerequisites
+An alternative to the standard installation, you can also use [syndesis](https://doc.syndesis.io/#syndesis).
+This tool, which is also used for building Syndesis itself, provides much more options to tune the installation.
```
Comment (kahboom) [inline content/quickstart.md]: "provides many more options"

File: content/quickstart.md [hunk 3]
```diff
@@ -48,43 +123,31 @@ $ minishift config set memory 8384
...
 ```
 
 In order to make it easy to run Syndesis on a cluster without requiring admin rights, Syndesis takes advantage of OpenShift's ability to use a [Service Account as an OAuth client](https://docs.openshift.org/latest/architecture/additional_concepts/authentication.html#service-accounts-as-oauth-clients). Before we create the app, we'll need to create this Service Account:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis/master/app/deploy/support/serviceaccount-as-oauthclient-restricted.yml
+$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis/master/install/support/serviceaccount-as-oauthclient-restricted.yml
 ```
 
 Deploy syndesis using the following command, replacing "syndesis-dev-restricted" with "syndesis-restricted" depending on the template
```
Comment (kahboom) [inline content/quickstart.md]: Let's capitalize Syndesis here

File: content/quickstart.md [hunk 4]
```diff
@@ -17,21 +17,96 @@ toc: true
 weight: 20
 ---
 
-<center>
-<iframe width="560" height="315" src="https://www.youtube.com/embed/9vaVMbsHJqk?rel=0" frameborder="0" allowfullscreen></iframe>
-</center>
+You can try out Syndesis very easily locally, too.
+All you need is a [Minishift](https://www.openshift.org/minishift/) installation which is available for all the  major operating systems (Linux, OS X and Windows).
+The following examples assume that you have Minishift installed and can be called with `minishift` from the command line (i.e. `minishift` is to be supposed to be findable in your search path)
```
[the remark this answers is not in the document]
Comment (kahboom) [reply content/quickstart.md]: Much better than an `iframe` with a reference to a YouTube video. Good job! 👍

File: content/quickstart.md [hunk 5]
```diff
@@ -48,43 +123,31 @@ $ minishift config set memory 8384
...
 
-* **Developer** : Use the template [`syndesis-dev-restricted`](https://raw.githubusercontent.com/syndesisio/syndesis/master/app/deploy/syndesis-dev-restricted.yml) which directly references Docker images without image streams. Then before building the images e.g. with `mvn fabric8:build` set your `DOCKER_HOST` environment variable to use the Minishift Docker daemon via `eval $(minishift docker-env)`. When new images are built you only need to delete the appropriate pod so that the new pod spinning up will use the freshly built image.
+* **User** : In case you only want to have the latest version of Syndesis on your local Minishift installation, use the template [`syndesis`](https://raw.githubusercontent.com/syndesisio/syndesis/master/app/deploy/syndesis.yml) which uses image stream referring to the published Docker Hub images. Minishift will update its images and trigger a redeployment when the images at Docker Hub changes. Therefore it checks every 15 minutes for a changed image. You do not have to do anything to get your application updated, except for waiting on Minishift to pick up new images.
```
Comment (kahboom) [inline content/quickstart.md]: "Therefore," or "As such," although of course this is not an English essay in college, I'm not sure why I'm correcting this. 😂  It's really perfect as is.

**Comment by kahboom** [comment]
Also none of this is a show-stopper, so I think merging wouldn't be the end of the world and we can make changes thereafter if it's urgent.

File: content/quickstart.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rhuss) [reply content/quickstart.md]: well the video was hopelessly outdated, so I removed it. Maybe another quickstart video would be nice, still ?

File: content/quickstart.md [hunk 6]
```diff
@@ -17,21 +17,96 @@ toc: true
...
+
+Then you reach the openshift login mask asked to login. Just use "developer" / "developer" as credentials
+
+![login](../images/login.png)
+
+Finally you are asked to grant OpenShift permissions to this account which you should accept
+
+![grant](../images/grant.png)
+
+Et voilà, welcome in Syndesisland !
```
[the remark this answers is not in the document]
Comment (rhuss) [reply content/quickstart.md]: Ça fait riens ;-)

File: layouts/index.html [hunk 7]
```diff
@@ -103,14 +103,13 @@ <h4>
     <div class="row align-items-center">
       <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 mb-2">
         <a href="http://openshift.org/"><img style="height: 10rem" src="images/openshift-logo.svg" alt="OpenShift"/></a>
-        <a href="https://kubernetes.io/"><img style="height: 8rem" src="images/kubernetes-logo.svg" alt="Kubernetes"/></a>
       </div>
       <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6">
         <h4>
-          Flexible Deployments on Kubernetes or OpenShift
+          Flexible Deployments on OpenShift
```
[the remark this answers is not in the document]
Comment (rhuss) [reply layouts/index.html]: Truely sad, but lets stay honest. Maybe we can come back to Kubernetes at some point in time (e.g. when the next rewrite happens, which according to @chirino is inevitable anyway ;-).

File: content/quickstart.md [hunk 8]
```diff
@@ -17,21 +17,96 @@ toc: true
...
+```
+
+Now you have now two ways available to install Syndesis:
+
+* [syndesis minishift](https://doc.syndesis.io/#syndesis-minishift) for setting up a Minishift Syndesis installation like described above
+* [syndesis install](https://doc.syndesis.io/#syndesis-install) for installing Syndesis to any OpenShift cluster.
+
+Please refer to the [Syndesis Developer Handbook](https://doc.syndesis.io/#syndesis) (SDH) for all the details and possible options. You can always use `syndesis --help` for get an online help or `syndesis --man` to open the corresponding chapter in the SDH.
+
+## Plain Minishift
```
[the remark this answers is not in the document]
Comment (rhuss) [reply content/quickstart.md]: another idea: "Minishift, the hard way". well, its not so hard anyway so lets pick "Vanilla"

File: content/quickstart.md (hunk 1 above)
Comment (rhuss) [reply content/quickstart.md]: no particular reason, maybe the only excuse is that we capitalized Everything Here In Germany ;-) (and maybe also a reference to the screenshot). But right, should be lowercase

File: content/quickstart.md (hunk 3 above)
Comment (rhuss) [reply content/quickstart.md]: good point, also `-restricted` is not anymore.

File: content/quickstart.md (hunk 5 above)
Comment (rhuss) [reply content/quickstart.md]: Well, my Grammarly checker allows both. I'm very lousy in english grammar and commas, so I typically trust it, but lets update here :-)

File: content/quickstart.md [hunk 9]
```diff
@@ -17,21 +17,96 @@ toc: true
...
+All you need is a [Minishift](https://www.openshift.org/minishift/) installation which is available for all the  major operating systems (Linux, OS X and Windows).
+The following examples assume that you have Minishift installed and can be called with `minishift` from the command line. So, `minishift` is to be supposed to be available in your search path, i.e. located in a directory contained in your `$PATH` environment variable (Linux, macOS) or in a directory from your system path (Windows)
 
-We aim to make it as simple as possible for users to try out Syndesis. If you want to try out locally on your laptop, follow the [Using Minishift]({{< relref "#using-minishift" >}}) instructions.
 
-- - -
+## Rocket launch
 
-## Using Minishift
+To make the installation super easy, on macOS and Linux you can use the following command for a complete installation of Syndesis on Minishift in a namespace `syndesis`:
```
[the remark this answers is not in the document]
Comment (rhuss) [reply content/quickstart.md]: added some explanation that it will be created if non-existant.

**Pull request merged**